### DISCUSSION TITLE
Can not connect to the target error

### DISCUSSION THREAD

**Discussion by ShuoShenDe** [foxglove/community#597] [Q&A]
**Description**
Hi~Dear foxglove team,
Happy new year. Here I try to debug the studio project. After I clone from GitHub and Run `corepack enable` and `yarn install`. Then I run it with vscode.
![Screenshot from 2023-01-04 09-28-52](https://user-images.githubusercontent.com/31400000/210514147-3dd53402-79b7-4961-87da-0de4a07561cf.png)
However, it gives me the error as follows:
![Screenshot from 2023-01-04 09-26-53](https://user-images.githubusercontent.com/31400000/210513876-c3807a3d-e89c-4342-a8ac-baebc8caf801.png)

What should I do to start debug? Thanks

- Version: 
- node -v
v16.18.1
npm -v
8.19.2
- Platform:
Ubuntu 18

**Steps To Reproduce**

Clone the studio from GitHub
Run `corepack enable` and `yarn install`
Then in vscode, run Attach to Chrome 
**Expected Behavior**
Get the error

**Comment by ShuoShenDe** [comment]
I might found the reason, this might beacuse it need chrome-debuggi, so run the command in terminial:
` google-chrome --remote-debuggi-port=9222`
Then you will see a new chrome window. This command can run, but it gives many errors, the breakpoints are still not working.
Is there any tutorial for setting development environment?

**Comment by defunctzombie** [comment]
Removing the bug label and moving this to a discussion. We use the bug label for issues with Studio features itself rather than your development environment.
